### ISSUE TITLE
CLNRest: Cannot create LN invoice with empty amount

### ISSUE THREAD

**Issue by matevz** [ZeusLN/zeus#2604]
### Describe the bug

Connecting to local core lightning node (CLNRest) and creating an invoice with empty amount yields `amount_msat: should be positive msat or 'any': invalid token '0'`.

### Reproduce

1. Configure core lightning wallet (CLNRest)
2. Lightning -> Receive
3. Leave amount empty
4. Create invoice

### ZEUS version

v0.9.2

### Node interface

Core Lightning (CLNRest)

### Network

Clearnet

### Device

Pixel 7a

### Device operating system

Android 15

### Log output

```shell
amount_msat: should be positive msat or 'any': invalid token '0'
```

**Comment by kaloudis** [comment]
Fix available for testing in ZEUS v0.9.4-beta1 https://github.com/ZeusLN/zeus/releases/tag/v0.9.4-beta1

**Comment by matevz** [comment]
I can confirm this works. Good job!
